src/vm/codeman.cpp

```diff
@@ -4300,7 +4318,7 @@ LPCWSTR ExecutionManager::GetJitName()
     
     if (NULL == pwzJitName)
     {
-        pwzJitName = MAKEDLLNAME_W(W("clrjit"));
+        pwzJitName = const_cast<LPWSTR>(MAKEDLLNAME_W(W("clrjit")));
```

Comment:
It seems we should rather change the signature of the CLRConfig::GetConfigValue to fix the problem. But I can see that it might not be trivial, so it would not make sense to make it part of this change. I'll create an issue for that.
But for now, could you rather do it differently - making the pwzJitName const again and casting the constness off at the call to the CLRConfig::GetConfigValue instead? We have several other places that do it that way and it would be easier later to fix them all.